test-requirements.txt

```diff
@@ -8,3 +8,6 @@ vcrpy-unittest
 openupgradelib @ git+https://github.com/OCA/openupgradelib.git
 unittest2 # For shopinvader test_controller, which inherits component
 odoo-test-helper
+
+odoo13-addon-base-rest @ git+https://github.com/OCA/rest-framework@refs/pull/71/head#subdirectory=setup/base_rest
+odoo13-addon-datamodel @ git+https://github.com/OCA/rest-framework@refs/pull/71/head#subdirectory=setup/datamodel
```

Comment:
@sebastienbeau datamodel is no more required. (datamodel bindings is provided by a dedicated addon now)

Reply:
Thansk @lmignon , I did the change